src/main/java/SearchElementInArray.java

```diff
@@ -1,39 +1,71 @@
 
+import java.util.Arrays;
 import java.util.Scanner;
 import java.util.Random;
 
 public class SearchElementInArray {
     public static void main(String[] args) {
-        // array size
+        // input array size in console
         Scanner s1 = new Scanner(System.in);
         System.out.print("Input array size: ");
         int arraySize = s1.nextInt();
+
         //Generate a random array of integers of provided size and print it
-        Random rd = new Random();
         int arr[] = new int[arraySize];
-        for (int n = 0; n < arr.length; n++) {
-            arr[n] = rd.nextInt();
-            System.out.println(arr[n]);
+        Random randomNumber = new Random();
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = randomNumber.nextInt(1000);
+            System.out.println(arr[i]);
         }
+
         // a number to search in the array
         Scanner s2 = new Scanner(System.in);
-        System.out.print("Input search number: ");
-        System.out.print(" ");
+        System.out.print("\nInput search number: ");
         int elementToSearch = s2.nextInt();
 
-        //int linear = linearSearch( arr[ arraySize],elementToSearch );
-    }
+        // method and measure time execution
+        long startLinear = System.nanoTime();
+        System.out.println("\nLinear search number is present in array: " + linearSearch(arr, elementToSearch));
+        long endLinear = System.nanoTime();
+        long linearTimeExecution = endLinear - startLinear;
 
-    public static int linearSearch(int arr[], int elementToSearch) {
+        long startBinary = System.nanoTime();
+        Arrays.sort(arr);
+        System.out.println("Binary search number is present in array: " + binarySearch(arr, elementToSearch));
+        long endBinary = System.nanoTime();
+        long binaryTimeExecution = endBinary - startBinary;
+
+        System.out.println("\nLinear Search takes " + linearTimeExecution + "ms");
+        System.out.println("Binary Search takes " + binaryTimeExecution + "ms");
+    }
 
+    public static boolean linearSearch(int[] arr, int i) {
         for (int index = 0; index < arr.length; index++) {
```

Comment:
It's OK to name loop variable as `index` but usually for such cases `i` letter is used.
As for the method arguments, as well as for local and other variables it would be better to name it according to its meaning, i.e. `int[] array, int searchValue`

Reply:
done